deploy-operator.sh

```diff
@@ -78,8 +78,11 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    test-network-function.com/subscription_name: '["nginx-operator-v0-0-1-sub"]'
   name: nginx-operator.v0.0.0
   namespace: placeholder
+  labels:
+    operators.coreos.com/nginx-operator.tnf: ""
```

Comment:
this replaces test-network-function.com/operator=target?

Reply:
yes

Reply:
need to add the variable for namespace, will fix

Reply:
how does autodiscovery work with a non-generic label like this?

Reply:
need to recheck this

Reply:
done